api/net/Areas/Editor/Controllers/WorkOrderController.cs

```diff
@@ -243,7 +243,6 @@ public async Task<IActionResult> RequestFileAsync([FromRoute] int locationId, [F
                         workOrder.Status = WorkOrderStatus.Failed;
                         workOrder.Note = "Remote file request to Kafka failed";
                         workOrder = _workOrderService.UpdateAndSave(workOrder);
-                        await _hub.Clients.All.SendAsync("WorkOrder", new WorkOrderMessageModel(workOrder, _serializerOptions));
```

Comment:
Not required.  A failed request should return an appropriate error response already.